generator/content.py

```diff
@@ -40,7 +40,7 @@ def _get_id_from_list(arr: Sequence[Any], item: str) -> int:
     return int(first.id)
 
 
-def generate_folders(namespaces: dict):
+def generate_folders(namespaces: dict, addl_owners: dict = {}):
```

Comment:
I think that for consistency this should be named `allowlist`
```suggestion
def generate_folders(namespaces: dict, allowlist: dict = {}):
```